elliott/elliottlib/cli/get_golang_report_cli.py

```diff
@@ -74,18 +71,69 @@ def get_golang_report_cli(runtime, ocp_versions, ignore_rhel, output):
                 v = golang_streams[b]
                 golang_streams_images[v].append(image_name)
 
+        # Analyze defined rpms
```

Comment:
This function has grown too big, can we split it up?

Reply:
Don't want to spend too much time on this, if it's okay would like to merge this change in and think of refactor down the line.